lib/super_parser.rb

```diff
@@ -0,0 +1,64 @@
+require 'webdrivers'
+require 'watir'
+require 'nokogiri'
+require 'pry'
+require 'json'
+
+require_relative 'super_parser/accounts'
+require_relative 'super_parser/transactions'
+
+class SuperParser
+  BASE_URL = 'https://my.fibank.bg/'.freeze
+
+  attr_accessor :browser, :account_details, :last_transactions
+
+  def initialize
+    web_connect
```

Comment:
initialize usually is not supposed to call all the logic. Better extract that calls of your methods to a separate methods like `execute`